tools/tools.go

```diff
@@ -338,22 +341,107 @@ func EnvmanClear(envstorePth string) error {
 	return nil
 }
 
-// EnvmanRun ...
+const enableSecretFilteringKey = "BITRISE_ENABLE_SECRET_FILTERING"
+
+// filteringEnabled returns true if enableSecretFilteringKey env presents with true value, false otherwise.
+func filteringEnabled(secrets []envmanModels.EnvironmentItemModel) bool {
+	for _, secret := range secrets {
+		key, value, err := secret.GetKeyValuePair()
+		if err != nil {
+			return false
+		}
+		if key == enableSecretFilteringKey && value == "true" {
```

Comment:
please merge if-s, `key == enableSecretFilteringKey && value == "true"` -> `err == nil && key == enableSecretFilteringKey && value == "true"` because the error is not used for anything else